pkg/engine/const.go

```diff
@@ -150,6 +150,7 @@ const (
 	kubernetesWindowsKubeletFunctionsPS1  = "k8s/windowskubeletfunc.ps1"
 	kubernetesWindowsCniFunctionsPS1      = "k8s/windowscnifunc.ps1"
 	kubernetesWindowsAzureCniFunctionsPS1 = "k8s/windowsazurecnifunc.ps1"
+	kubernetesWindowsOnRestartPS1         = "k8s/on-restart.ps1"
```

Comment:
can you prefix with `windows...` for consistency?

Reply:
done